_posts/2022-05-05-8.0.0-release.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Apache Arrow 8.0.0 Release"
+date: "2022-05-05 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 8.0.0 release. This covers
+over 3 months of development work and includes [**586 resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 7.0.0 release, Kun Liu, Raphael Taylor-Davies Xudong Wang, Yijie Shen
+and Liang-Chi Hsieh have been invited to be committers.
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
+
```

Comment:
```suggestion
## Go notes

### Bug fixes

* parquet_reader / parquet_schema no longer crash [ARROW-15509](https://github.com/apache/arrow/pull/12303)
* Base64 encoding of origin Arrow schema properly uses padding and decodes both with or without the padding in pqarrow.getOriginSchema [ARROW-15544](https://github.com/apache/arrow/pull/12679)
* ipc.Writer no longer includes unnecessary offsets when encoding sliced arrays [ARROW-15715](https://github.com/apache/arrow/pull/12453)
* Use base64.StdEncoding for Arrow Flight Basic Auth middleware for proper encoding/decoding [ARROW-15772](https://github.com/apache/arrow/pull/12503)
* Fix panic during concurrent compression of ipc body buffers due to negative WaitGroup counter [ARROW-15792](https://github.com/apache/arrow/pull/12518)
* Fix memory leak in pqarrow.NewColumnWriter with nested structures [ARROW-15946](https://github.com/apache/arrow/pull/12641)
* ipc.FileReader no longer leaks memory when using ZSTD compression [ARROW-16163](https://github.com/apache/arrow/pull/12857)

### Enhancements

#### Flight RPC

* *Breaking Change* gRPC version is updated and Flight Server creation has been simplified. Flight servers must now embed flight.BaseFlightServer [ARROW-15418](https://github.com/apache/arrow/pull/12233)
* You can now provide a full net.Listener as an alternative to just providing an address to bind to when creating a Flight server [ARROW-16082](https://github.com/apache/arrow/pull/12768)

#### Parquet

* 'unpack_bool', sum_float64, and bitmap handling functions have been given optimized assembly implementation for Arm64 NEON [ARROW-15440](https://github.com/apache/arrow/pull/12398), [ARROW-15742](https://github.com/apache/arrow/pull/12502), [ARROW-15995](https://github.com/apache/arrow/pull/12687)
* Go Parquet handling has been simplified to only need io.ReadSeeker instead of a ReadAtSeeker interface [ARROW-15963](https://github.com/apache/arrow/pull/12658)
* BitSetRunReader and helper functions have been lifted to internal/bitutils package to share between arrow and parquet implementations [ARROW-15950](https://github.com/apache/arrow/pull/12926)
* Parquet Reader now properly obeys the buffer size read property for buffered streams [ARROW-16187](https://github.com/apache/arrow/pull/12876)
* parquet NewBufferedReader no longer panics [ARROW-16283](https://github.com/apache/arrow/pull/12960)

#### Arrow

* Go Arrow library now supports Dictionary Arrays [ARROW-3039, ARROW-9378](https://github.com/apache/arrow/pull/12158)
* array.ArrayEqual, array.ArrayApproxEqual have been renamed to array.Equal and array.ApproxEqual. Aliases are provided to avoid breaking existing code which will be removed in v9. [ARROW-5598](https://github.com/apache/arrow/pull/12877)
* Custom cpu discovery package replaced with using golang.org/x/sys/cpu [ARROW-16193](https://github.com/apache/arrow/pull/12764)
* *Breaking Change* array.Interface, array.Record, array.Table, etc. were deprecated in v7 in favor of arrow.Array, arrow.Record, etc. The deprecated aliases have been removed in v8 [ARROW-16192](https://github.com/apache/arrow/pull/12960)

#### CI

* staticcheck is now run as part of CI to lint the Go code [ARROW-15296](https://github.com/apache/arrow/pull/12540)
* Travis builds on Arm64 for Go are no longer allowed to fail [ARROW-15400](https://github.com/apache/arrow/pull/12214)

```